packages/react/src/components/Portal/index.ts

```diff
@@ -1,22 +1,28 @@
-/**
- * Copyright IBM Corp. 2016, 2023
- *
- * This source code is licensed under the Apache-2.0 license found in the
- * LICENSE file in the root directory of this source tree.
- */
```

Comment:
Did you remove the copyright info accidentally?

Reply:
Yeah, sorry i missed them when copying from the js file